src/features/RedirectPage/RedirectPage.tsx

```
@@ -10,179 +9,139 @@ import {
-        }
+        router.push("/service-plans");
       }
-    };
 
-    subscribeAndRedirect();
+      return;
+    }
+
+    // Find Service Offering with a Subscription and Redirect them To the Corresponding Instances List Page
+    const serviceOfferingsObj = serviceOfferingsData?.reduce(
+      (acc, curr) => ({ ...acc, [curr.productTierID]: curr }),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)